modeltranslation/utils.py

```diff
@@ -163,12 +168,11 @@ def fallbacks(enable=True):
     processing or check if there is a value for the current language (not
     knowing the language)
     """
-    current_enable_fallbacks = settings.ENABLE_FALLBACKS
-    settings.ENABLE_FALLBACKS = enable
+    set_enable_fallbacks(enable)
     try:
         yield
     finally:
-        settings.ENABLE_FALLBACKS = current_enable_fallbacks
+        set_enable_fallbacks(None)
```

Comment:
```suggestion
        set_enable_fallbacks(None)
```

same here, need's to be reverted to previous.